samples/minimal.yaml

```diff
@@ -0,0 +1,99 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This sample configuration provides the minimum configuration required by the DPT scripts.
+# Audit resources will be created locally in the project.
+
+overall:
+  organization_id: '12345678'           # Replace this with your Organization ID, or remove to setup without an organization.
+  billing_account: 000000-000000-000000 # Replace this with your Billing Account.
+  domain: domain.com                    # Replace this with your domain.
+
+projects:
+- project_id: my-project
+  owners_group: my-project-owners@my-domain.com      # Replace this with the owners group for this project.
+  auditors_group: some-auditors-group@my-domain.com  # Replace this with your auditors group.
+  audit_logs:
+    logs_bq_dataset:
+      properties:
+        location: US
+        name: audit_logs
+  resources:
+    gke_clusters:
+    - properties:
+        clusterLocationType: Regional
+        region: us-central1
+        cluster:
+          name: clusterName
+          description: clusterDescription
+          network: "projects/my-project/global/networks/default"
+          subnetwork: "projects/my-project/regions/us-central1/subnetworks/default"
+          nodeConfig:
+            oauthScopes: 
+              - "https://www.googleapis.com/auth/devstorage.read_only"
+              - "https://www.googleapis.com/auth/logging.write"
+              - "https://www.googleapis.com/auth/monitoring"
+              - "https://www.googleapis.com/auth/servicecontrol"
+              - "https://www.googleapis.com/auth/service.management.readonly"
+              - "https://www.googleapis.com/auth/trace.append"
+          initialNodeCount: 1
+          locations:
+            - us-central1-c
+            - us-central1-b            
+    gke_workloads:
+    - cluster_name: "clusterName"
+      properties:
+        apiVersion: "extensions/v1beta1"
+        kind: Deployment
+        metadata:
+          name: dicom-adapter
+          namespace: "default"
+          labels:
+            app: "dicom-adapter"
+        spec:
+          replicas: 1
+          template:
+            metadata:
+              labels:
+                app: dicom-adapter
+            spec:
+              containers:
+                - name: dicom-import-adapter
+                  image: gcr.io/path-to/dicom-import-adapter
+                  ports:
+                    - containerPort: 2575
+                      protocol: TCP
+                      name: "port"
+                  command:
+                    - "/import/bin/import"
+                    - "--dimse_aet=IMPORTADAPTER"
+                    - "--dimse_port=2575"
+                    - "--dicomweb_addr=https://healthcare.googleapis.com/v1beta1/projects/my-project/locations/us-central1-c/datasets/dataset/dicomStores/dicomStores/dicomWeb"
```

Comment:
This uses the old version of these parameters, please upgrade to the new version (see the current readme)

Reply:
oh, i must be using old version of adapter for tests
done